core/arch/arm/tee/entry_std.c

```diff
@@ -221,6 +222,12 @@ static void entry_invoke_command(struct thread_smc_args *smc_args,
 	struct tee_dispatch_invoke_command_out out;
 	struct optee_msg_param *params = OPTEE_MSG_GET_PARAMS(arg);
 
+#ifdef CFG_TEE_BENCHMARK
+	if (params[TEE_BENCH_DEF_PARAM].u.tmem.buf_ptr)
+		tee_add_timestamp((void *)(uintptr_t)
+			params[TEE_BENCH_DEF_PARAM].u.tmem.buf_ptr,
+			TEE_BENCH_CORE);
+#endif
```

Comment:
could simplify to this ?
```
+#ifdef CFG_TEE_BENCHMARK
+		tee_benchmark_add_timestamp(params);
+#endif
```
and hide all `params` handling inside  `tee_add_timestamp()`.
and even hide the `#ifdef` inside  `tee_add_timestamp()`.